### DISCUSSION TITLE
Editor behavior when creating midi message

### DISCUSSION THREAD

**Discussion by jetstobrazil** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#110] [Editor App Feature Requests]
When creating a message (like.... CC message for example), I'd like to click on the Channel text box, type it, press tab, type info, etc etc, and then at the end, I wish I could just press ENTER to submit (confirm) it. but It won't do it. 

I think it's a very easy fix, but it would save myself A LOT of time.

**Comment by simonaglover95** [comment]
Hi @jetstobrazil There are orderly tab-to-move controls in message editing windows now - although it first goes through all the MIDI output options before the cancel/confirm buttons - It works great :)
